agent/php_nrini.c

```diff
@@ -1891,6 +1891,42 @@ static PHP_INI_MH(nr_custom_events_max_samples_stored_mh) {
   return SUCCESS;
 }
 
+#define DEFAULT_WORDPRESS_HOOKS_OPTIONS "all_callbacks"
+static PHP_INI_MH(nr_wordpress_hooks_options_mh) {
+  nrinistr_t* p;
+
+  char* base = (char*)mh_arg2;
+
+  p = (nrinistr_t*)(base + (size_t)mh_arg1);
+
+  (void)mh_arg3;
+  NR_UNUSED_TSRMLS;
+
+  if (NEW_VALUE_LEN > 0) {
```

Comment:
If this INI value has an empty string associated with it then from the logical below the code will default to the "all_callbacks" option.  Would it make sense to store this as `p->value` instead of leaving it a NULL pointer (initialized globals are 0 I believe)?

Reply:
Testing shows:
- When `newrelic.framework.wordpress.hooks.options` is not in the INI, `newrelic.framework.wordpress.hooks.options` is set to `all_callbacks` (output from `phpinfo()`) (the default is set [here](https://github.com/newrelic/newrelic-php-agent/blob/97edd07795b66bdcc74920fd321f18e750391f7e/agent/php_nrini.c#L2246))
- When `newrelic.framework.wordpress.hooks.options` is in the INI, and is empty (`newrelic.framework.wordpress.hooks.options=""` or `newrelic.framework.wordpress.hooks.options=`), `newrelic.framework.wordpress.hooks.options` is set to `no value` (output from `phpinfo()`)
- When `newrelic.framework.wordpress.hooks.options` is in the INI, and is set to invalid value (neither `all_callbacks`, nor `plugin_callbacks`, nor `threshold`), `newrelic.framework.wordpress.hooks.options` is set to that invalid value (output from `phpinfo()`) but a warning is logged in agent's log file.

Reply:
@mfulb I made `newrelic.framework.wordpress.hooks.options` modify handler more robust in cd9745eaa15c86a4ca332f6fffcd4cdbd46f38ca.